### DISCUSSION TITLE
Is there anyway to delete the selected collections?

### DISCUSSION THREAD

**Discussion by WeatherAnchor** [isar/isar#504] [Help] [answered]
I want to delete a collection because I changed the field type and even after generating the file with `flutter pub run build_runner build`, its causing an error. 
Error: `IsarError (IsarError: SchemaError: Property "startTime" already exists but has a different type)`
In **Hive** database you can do something like this: `await Hive.deleteBoxFromDisk('mytestCollections');`. 
I was wondering if there is something similar to this? Or any suggestions are appreciated.

**Comment by simc** [comment, marked as answer]
The database on your device still contains the old schema. Either delete the database or Just rename the new collection. 

Generally collections are deleted if you no longer specify them in the `schemas` when you open Isar.

If you don't want to change the Dart name of your collection, use the `@Name()` annotation.

**Reply by WeatherAnchor** [reply]
Thanks for your answer. `Either delete the database or Just rename the new collection.` How do I delete the database?

**Reply by simc** [reply]
Depends on your Isar version. For the newest one delete `path/yourIsarName.isar`
